src/Filesystem/Folder.php

```diff
@@ -183,6 +201,12 @@ public function read($sort = true, $exceptions = false, $fullPath = false)
             return [$dirs, $files];
         }
 
+        if (!is_bool($sort) && array_key_exists($sort, $this->_fsorts)) {
```

Comment:
IMO isset() should be preferred over array_key_exists() as null values are not relevant here.